test/parallel/test-os.js

```diff
@@ -130,12 +130,35 @@ switch (platform) {
       const actual = interfaces['Loopback Pseudo-Interface 1'].filter(filter);
       const expected = [{ address: '127.0.0.1', netmask: '255.0.0.0',
                           mac: '00:00:00:00:00:00', family: 'IPv4',
-                          internal: true }];
+                          internal: true, cidr: '127.0.0.1/8' }];
       assert.deepStrictEqual(actual, expected);
       break;
     }
 }
 
+// assert cidr prop present all interface info
+function flatten(arr) {
+  const _flatten = function(p, c) {
+    if (Array.isArray(c)) p = p.concat(c.reduce(_flatten, []));
+    else p.push(c);
+    return p;
+  };
+
+  return arr.reduce(_flatten, []);
+};
+const netmaskToCIDRSuffixMap = new Map(Object.entries({
```

Comment:
I like the use of `entries` but it will exclude this from backporting to `v6.x`, and it's a nice feature.
So be ready to [backport](https://github.com/nodejs/node/blob/master/doc/guides/backporting-to-release-lines.md).